examples/demo/site/modules/admin/www/README.adoc

```diff
@@ -0,0 +1,169 @@
+= HAL-browser
```

Comment:
Is it ok to embed the html5 HAL browser within ROC ? related to licence?
I asked because sometime with drupal there are 3rd party JS lib to download manually.
i.e not shipped with the related drupal module.

so I guess there may be sometime licensing issue.